app/src/main/res/layout/activity_description.xml

```diff
@@ -224,7 +224,7 @@
                             android:layout_height="wrap_content"
                             android:layout_marginLeft="20dp"
                             android:fontFamily="@font/armata"
-                            android:onClick="onClick"
```

Comment:
Ah, OK, that makes sense
You've changed the name in the Java file

Reply:
Alright, here you can see it in the XML file! 👍